hr_timesheet_date_order_desc/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Timesheets Date Order Desc",
+    "summary": "Add new timesheet entries to the top of the list and "
+    "order by date descending",
+    "version": "17.0.1.0.0",
+    "author": "Lansana Barry Sow, APSL-Nagarro, Odoo Community Association (OCA)",
```

Comment:
Don't put contributor names, just companies. The names are put in CONTRIBUTORS.md.

```suggestion
    "author": "APSL-Nagarro, Odoo Community Association (OCA)",
```

Reply:
@pedrobaeza , I have already applied the change.